lib/util/git/behind-base-branch-cache.ts

```diff
@@ -1,20 +1,45 @@
+import { logger } from '../../logger';
 import { getCache } from '../cache/repository';
+import { getBranchCommit } from '.';
```

Comment:
don't we have circular references now? that needs to be fixed, as it can lead to unexpected runtime behavior.